Exercise4/app.js

```diff
@@ -0,0 +1,103 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const fs = require('fs');
+let app = express();
+app.use(bodyParser.json());
+
+//See all tasks in the TODO list
+app.get('/tasks', (req, res) => {
+    res.send(readJsonFile());
```

Comment:
Better read the whole file when the server starts instead of reading it on every request